classes/renderer/multimedias.php

```diff
@@ -26,13 +26,17 @@ public function build()
             $this->set_value($this->getValueFromInstance($item));
         }
 
-        return (string) \View::forge('novius_renderers::multimedias/inputs', array(
+        $template = $this->template ?: $this->fieldset()->form()->get_config('field_template', "<tr><th>{label} :</th> <td>{field}</td></tr>");
```

Comment:
By setting `"<tr><th>{label} :</th> <td>{field}</td></tr>"` as default template you introduce a breaking change that will break existing renderers that are not in a `<table>` (like in the CRUD's accordion menu).

I would have set `{field}` as default template.